docs/ref/models/querysets.txt

```diff
@@ -2470,11 +2470,6 @@ be in conflict must be provided.
 Enabling the ``ignore_conflicts`` parameter disables setting the primary key on
 each model instance (if the database normally supports it).
 
-.. versionchanged:: 5.0
-
-    In older versions, enabling the ``update_conflicts`` parameter prevented
-    setting the primary key on each model instance.
-
```

Comment:
Please don't move this down.

Reply:
I don't think I understand what you mean by that. Where do I put it?